content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -68,6 +45,14 @@ There are several sampling types to choose from. Choosing the best sampling type
 |  `Bilinear` | The `Bilinear` filter smooths out the image by requesting four points in a grid around a requested pixel and then performing interpolation between the points.<br><br>Because this uses linear interpolation between 4 points, bilinear filtering can cause noticeable plus-shaped artifacts in the smoothed data. `Bilinear` sampling is the best general-purpose choice, as it balances performance and quality. | 4x | Good | ![Image Gradient using bilinear interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bilinear.png) |
 |  `Bicubic` | The `Bicubic` filter smooths out the image by requesting sixteen points in a grid around a requested pixel and then performing Catmull-Rom interpolation between the points.<br><br>Because this uses non-linear interpolation, there are no noticeable plus-shaped artifacts in the smoothed data. `Bicubic` sampling is the best choice when quality is needed over performance. | 16x | Great | ![Image Gradient using bicubic interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bicubic.png) |
 
+### Creating a new image
+
+To create a new image, press the **Create New Image...** button. You will be prompted for an image width and height in pixels, then for a location to save the image. If the image is saved into a source asset directory that is used by the project, the Image Gradient will automatically populate the **Image Asset** field with the saved image.
```

Comment:
Maybe worth linking to `https://www.o3de.org/docs/user-guide/assets/pipeline/scan-directories/` if the reader is curious why it needs to be in a directory used by the project.

Reply:
Added the link for both Image Gradient and Terrain Macro Material.